client/wordpress-com.js

```diff
@@ -208,16 +207,11 @@ if ( config.isEnabled( 'manage/drafts' ) ) {
 }
 
 if ( config.isEnabled( 'reader' ) ) {
-	readerPaths = [
-		'/',
-		'/read'
-	];
-
 	sections.push( {
-		name: 'reader',
-		paths: readerPaths,
-		module: 'reader',
-		secondary: true,
+		name: 'reader-full-post',
+		paths: [ '/read/feeds/[^\\/]+/posts/[^\\/]+', '/read/blogs/[^\\/]+/posts/[^\\/]+' ],
```

Comment:
ya. i think we could use https://www.npmjs.com/package/path-to-regexp like `page` does. Then we'd support the same syntax as the rest of our routes.